### PR TITLE
User defined indices list for sample kr

### PR DESCRIPTION
**Pull request by caglayantuna**
As we discussed in #349 , I updated sample_khatri_rao function to allow users to input their own indices list.

### CODE REVIEW

File: tensorly/decomposition/_cp.py [hunk 1]
```diff
@@ -446,6 +446,9 @@ def sample_khatri_rao(matrices, n_samples, skip_matrix=None,
     skip_matrix : None or int, optional, default is None
         if not None, index of a matrix to skip
 
+    indices_list : list, default is None
+        if None, random indices will be created
```
Comment (JeanKossaifi) [inline tensorly/decomposition/_cp.py:450]: Can we add a longer description, e.g. what is selected and range of the indices?

**Review by JeanKossaifi — COMMENTED** [review]
Looks good - I'd maybe just add a bit more info in the docstring.

**Review by JeanKossaifi — COMMENTED** [review]
I'd just detail more what is expected for that list of indices and what exactly it does but looks good to me otherwise!

File: tensorly/decomposition/_cp.py (hunk 1 above)
Comment (caglayantuna) [reply tensorly/decomposition/_cp.py:451]: I added some information, I hope it is ok for you

File: tensorly/decomposition/_cp.py [hunk 2]
```diff
@@ -446,6 +446,10 @@ def sample_khatri_rao(matrices, n_samples, skip_matrix=None,
     skip_matrix : None or int, optional, default is None
         if not None, index of a matrix to skip
 
+    indices_list : list, default is None
```
Comment (JeanKossaifi) [inline tensorly/decomposition/_cp.py:449]:
```suggestion
    indices_list : list, default is None
        Contains, for each matrix in `matrices`, a list of indices of rows to sample.
        if None, random indices will be created
```
Slight suggestion for clarity

**Comment by JeanKossaifi** [comment]
Looks good thanks @caglayantuna  - just added a small suggestion.

**Comment by JeanKossaifi** [comment]
Looks good to me, merging!

**Pull request merged**